test/e2e/deprecated_e2e_test.go

```diff
@@ -1,6 +1,137 @@
 package e2e
 
-// TODO
+import (
+	"context"
+	_ "embed"
+	"time"
 
-// v1beta1 CRD in installplan fails
-// v1beta1 RBAC in an installplan fails
+	"github.com/operator-framework/operator-lifecycle-manager/test/e2e/ctx"
+
+	"github.com/blang/semver/v4"
+	. "github.com/onsi/ginkgo"
+	"github.com/onsi/ginkgo/extensions/table"
+	. "github.com/onsi/gomega"
+	operatorsv1alpha1 "github.com/operator-framework/api/pkg/operators/v1alpha1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+//go:embed testdata/deprecated/fakeCRD.yaml
+var fakeCRD string
+
+//go:embed testdata/deprecated/fakeRBAC.yaml
```

Comment:
this results in the test panicking if it's called directly

Reply:
hmm, we could sidestep that issue by generating or hardcoding the manifest instead. IIUC, we really only need a manifest containing some metadata to ensure we handle non-existent GVKs; e.g.

```yaml
apiVersion: my.group/v1
kind: MyKind
metadata:
  name: my.thing
```

Reply:
this makes sense but the problem is we don't support arbitrary kinds, so instead of testing the behavior we are interested in we get back `v1alpha1.ErrInvalidInstallPlan`